js/models/stock.js

```diff
@@ -1,12 +1,36 @@
 // define your Stock model here
 function Stock(symbol) {
-  this.symbol = symbol;
+  this.symbol = document.getElementsByTagName("Symbol");
   this.numShares = 0;
+  this.name = document.getElementsByTagName("Name");
+  this.price = document.getElementsByTagName("LastPrice");
 
   // your code to search the API and return a Stock object here
 
 }
 
 Stock.prototype = {
-  totalValue: function() { return this.price * this.numShares; }
+  totalValue: function() {
+    return this.price * this.numShares;
+    },
+
+  fetchName: function(callback) {
+    $.ajax({
+      type: "GET",
+      dataType: "json",
+      url: "http://dev.markitondemand.com/Api/v2/Quote/Name"
+    }).done(function(response) {
+      stockModel.loadStock(response);
```

Comment:
this information for the ajax call itself looks good, but what does the response really giving you.